documentation/source/users/rmg/installation/virtualMachineSetup.rst

```diff
@@ -4,6 +4,40 @@
 Setting up a Linux Virtual Machine from Windows
 ************************************************
 
+Checking if Virtualization Technology is Enabled
+=================================================
+
+While most PCs have CPUs that support running virtual machines (i.e. virtualization technology), it is possible that
+this feature has been disabled in your BIOS settings. If this is the case, then you will have to enable this technology
+in your BIOS settings.
+
+First, check to see if virtualization technology is currently enabled on your computer by opening the Task Manager in
+Windows. Click on the ``Performance`` tab--from here you can see if virtualization technology is enabled.
+
+.. image:: images/Virtualization_Enabled.png
+    :align: center
+
+If enabled, you can continue on with installing a virtual machine on your PC. If virtualization is currently disabled,
+though, you will have to enable this from the BIOS setting on your computer. How to do this varies from PC to PC (we
+recommend doing a quick google search for your make/model, as there are many instructions for this online), but the
+basic steps are as follows:
+
+1. Restart the PC. As soon as the PC turns back on, enter the BIOS settings (this usually involves pressing the ``F2``
+or ``F12`` keys, but will depend on your make/model). If you see the computer loading Windows then you have missed the
+opportunity, and should restart the PC to try again. If Secure Boot is enabled on your PC then there may be additional
+steps to reaching the BIOS settings.
+
+2. From the BIOS settings, find the section on virtualization, and enable the virtualization technology. Save these
+changes and restart the PC.
+
+For more information, please see online tutorials like
+`this one here <https://support.bluestacks.com/hc/en-us/articles/115003910391-How-can-I-enable-virtualization-VT-on-my-PC->`_.
+
+**Note that changing your BIOS settings can be risky, so follow online tutorials carefully**. If you are unsure of what
+to do even after looking for online tutorials, consider following the instructions for installing RMG inside the Linux
+subsystem in Windows.
```

Comment:
Great

Reply:
I also added a link here to these instructions